tests/model_registry/model_catalog/conftest.py

```diff
@@ -4,11 +4,14 @@
 from kubernetes.dynamic import DynamicClient
 
 from ocp_resources.config_map import ConfigMap
+from ocp_resources.model_registry_components_platform_opendatahub_io import ModelRegistry
```

Comment:
why not `from ocp_resources.model_registry_modelregistry_opendatahub_io import ModelRegistry` ?